src/test/java/com/contact/application/controller/ContactControllerTest.java

```diff
@@ -0,0 +1,136 @@
+package com.contact.application.controller;
+
+import com.contact.application.model.Contact;
+import com.contact.application.service.TestUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ContactControllerTest {
+    private TestUtils testUtils;
+    private GetController getControllerMock = mock(GetController.class);
```

Comment:
Can you use an annotation `@Mock` to automatically mock these?

Reply:
Sure, will address.